src/main/java/com/cinema/dao/ShoppingCartDao.java

```diff
@@ -0,0 +1,12 @@
+package com.cinema.dao;
+
+import com.cinema.model.ShoppingCart;
+import com.cinema.model.User;
+
+public interface ShoppingCartDao {
```

Comment:
let's make one more commit here and see if these classes disappear

Reply:
done